test/rate_limiter_test.cc

```diff
@@ -172,4 +172,123 @@ TEST_F(RateLimiterTest, DistributionSamplingRateLimiterImplSchedulingTest) {
   EXPECT_TRUE(rate_limiter->tryAcquireOne());
 }
 
+// TODO(oschaaf): once we have hr sleep, test at a higher res.
+class LinearRampingRateLimiterImplTest : public Test {
+public:
+  std::vector<int64_t> getAcquisitionTimings(const Frequency frequency,
+                                             const std::chrono::seconds duration) {
+    Envoy::Event::SimulatedTimeSystem time_system;
+    std::vector<int64_t> aquisition_timings;
+    LinearRampingRateLimiterImpl rate_limiter(time_system, duration, frequency);
+    auto total_ms_elapsed = 0ms;
+    const auto clock_tick = 1ms;
+    auto last_acquisition_timestamp = 0ms;
```

Comment:
What is last_acquisition_timestamp used for?

Reply:
Removed in 6329377a31b0b35828d81c2e2a22cfd8f3943e8a